main/acle.md

```diff
@@ -8636,6 +8641,397 @@ This is because the ACLE intrinsic calls do not imply a particular
+tuple types `svboolx2_t` and `svboolx4_t`.
+
+These are opaque tuple types that can be accessed using the SVE intrinsics
+`svsetN`, `svgetN` and `svcreateN`. `svundef2` and `svundef4` are also extended
+to work with `svboolx2_t` and `svboolx4_t`.  e.g.
+
+``` c
+    svbool_t svget2[_b](svboolx2_t tuple, uint64_t imm_index);
+    svboolx2_t svset2[_b](svboolx2_t tuple, uint64_t imm_index, svbool_t x);
+    svboolx2_t svcreate2[_b](svbool_t x, svbool_t y);
+    svboolx2_t svundef2_b();
+```
+
+#### ADDQV, FADDQV
+
+Unsigned/FP add reduction of quadword vector segments.
+
+``` c
+  // Variants are also available for:
+  // _s8, _s16, _u16, _s32, _u32, _s64, _u64,
+  // _f16, _f32, _f64
+  uint8x16_t svaddqv[_u8](svbool_t pg, svuint8_t zn);
+  ```
+
+#### ANDQV, EORQV, ORQV
+
+Reduction of quadword vector segments.
+
+``` c
+  // Variants are also available for:
+  // _s8, _u16, _s16, _u32, _s32, _u64, _s64
+  uint8x16_t svandqv[_u8](svbool_t pg, svuint8_t zn);
+  uint8x16_t sveorqv[_u8](svbool_t pg, svuint8_t zn);
+  uint8x16_t svorqv[_u8](svbool_t pg, svuint8_t zn);
+  ```
+
+#### DUPQ
+
+Broadcast indexed element within each quadword vector segment.
+
+``` c
+   // Variants are also available for:
+   // _s8, _u16, _s16, _u32, _s32, _u64, _s64
+   // _bf16, _f16, _f32, _f64
+   svuint8_t svdup_laneq[_u8](svuint8_t zn, uint64_t imm_idx);
+   ```
+
+#### EXTQ
+
+Extract vector segment from each pair of quadword segments.
+
+``` c
+   // Variants are also available for:
+   // _s8, _s16, _u16, _s32, _u32, _s64, _u64
+   // _bf16, _f16, _f32, _f64
+   svuint8_t svextq[_u8](svuint8_t zdn, svuint8_t zm, uint64_t imm);
+   ```
+#### LD1D, LD1W
+
+Contiguous zero-extending load to quadword (single vector).
+
+``` c
+   // Variants are also available for:
+   // _u32, _s32
+   svfloat32_t svld1uwq[_f32](svbool_t, const float32_t *ptr);
+   svfloat32_t svld1uwq_vnum[_f32](svbool_t, const float32_t *ptr, int64_t vnum);
+
+
+   // Variants are also available for:
+   // _u64, _s64
+   svfloat64_t svld1udq[_f64](svbool_t, const float64_t *ptr);
+   svfloat64_t svld1udq_vnum[_f64](svbool_t, const float64_t *ptr, int64_t vnum);
+   ```
+
+#### LD1Q
+
+Gather Load Quadword.
+
+``` c
+   // Variants are also available for:
+   // _u8, _u16, _s16, _u32, _s32, _u64, _s64
+   // _bf16, _f16, _f32, _f64
+   svint8_t svld1q_gather[_u64base]_s8(svbool_t pg, svuint64_t zn);
+   svint8_t svld1q_gather[_u64base]_offset_s8(svbool_t pg, svuint64_t zn, int64_t offset);
+   svint8_t svld1q_gather_[u64]offset[_s8](svbool_t pg, const int8_t *base, svuint64_t offset);
+
+
+   // Variants are also available for:
+   // _u16, _u32, _s32, _u64, _s64
+   // _bf16, _f16, _f32, _f64
+   svint16_t svld1q_gather_[u64]index[_s16](svbool_t pg, const int16_t *base, svuint64_t index);
+   svint8_t svld1q_gather[_u64base]_index_s8(svbool_t pg, svuint64_t zn, int64_t index);
+   ```
+
+#### LD2Q, LD3Q, LD4Q
+
+Contiguous load two, three or four quadword structures.
```

Comment:
missing comma after "three" - always use the oxford/serial comma to reduce ambiguity https://confluence.arm.com/pages/viewpage.action?pageId=345213178

Reply:
Fixed in #326